From f2184e7573e705bad408cf1422599b967773d48e Mon Sep 17 00:00:00 2001
From: Quirin Gylstorff <quirin.gylstorff@siemens.com>
Date: Thu, 16 Nov 2023 16:40:19 +0100
Subject: [PATCH 4/4] d/swupdate-www.install: Fix path for debian-bullseye

Signed-off-by: Quirin Gylstorff <quirin.gylstorff@siemens.com>
---
 debian/swupdate-www.install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/swupdate-www.install b/debian/swupdate-www.install
index 7e3d95c6..9c6113cc 100644
--- a/debian/swupdate-www.install
+++ b/debian/swupdate-www.install
@@ -1,5 +1,5 @@
 web-app/css          /usr/share/swupdate/www
 web-app/images       /usr/share/swupdate/www
-index.html           /usr/share/swupdate/www
+web-app/index.html   /usr/share/swupdate/www
 web-app/js           /usr/share/swupdate/www
 web-app/node_modules /usr/share/swupdate/www
-- 
2.42.0

